piperider_cli/cli.py

```diff
@@ -145,19 +153,23 @@ def run(**kwargs):
         workspace.generate_report()
 
 
-@cli.command(short_help='Show report')
-@click.option('--input', default=None, type=click.Path(exists=True), help='Path of json report file')
+@cli.command(short_help='Generate a report.')
+@click.option('--input', default=None, type=click.Path(exists=True), help='Specify the raw result file.')
```

Comment:
For new user, will they know what a raw result file is?
Maybe mention `.piperider/outputs`  (similar to how documentation site explains it)